docs/workflow/ci/failure-analysis.md

```diff
@@ -123,6 +123,11 @@ To unconditionally bypass the build analysis check (turn it green), you can add
 
 The `Build Analysis` requests are sent to a queue. In certain scenarios, this queue can have many items to process and it can take a while for the status to be updated. If you do not see the status getting updated, be patient and wait at least 10 minutes before investigating further.
 
+While most failures can be matched via known issues, a few failures modes cannot be matched currently and it is valid to suppress them manually. Please use the following reasons to suppress them:
+
+- `/ba-g doc changes only` - build analysis won't turn green for PRs that contain documentation .md file changes only.
```

Comment:
How important is it that we use these exact phrases? Is that just for better telemetry?